gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -230,6 +230,10 @@ services:
 # the type as "none"
 analytics:
   type: elasticsearch # or none
+#  logs:
+     # Maximum number of APIs and applications a query to the analytics database can contain when searching for platform logs.
+     # Default value of 250, means the query can contain at most 250 apis and 250 applications.
+#    maxItemsNumberInPlatformLogQuery: 250
```

Comment:
As discussed, the `max_clause_count` seems to be a limitation applied per filter, meaning that you can have a `apis` filter with up to 1024 and an `applications` filter with up to 1024. Maybe we should change a bit the current implementation?
Regarding the default value of 250, elastic sets it to 1024 by default. Should we align it?

Reply:
I've decided to remove all the limitation part from this PR.
We already had the same kind of issue for analytics: https://gravitee.atlassian.net/browse/APIM-1686
And potential maxClauseCount issue has not been taken into account.

On top of that, limit the number of API and applications could be seen as a big regression for users and I finally don't think that it is the kind of decision we can make at support while fixing a bug.

Reply:
Agree